handlers/mapping_handlers.go

```diff
@@ -327,16 +332,16 @@ func (env *Env) DeleteMappingHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if *sessionMember.Pending {
-		errMsg := "Cannot remove users from conversation while invitation is pending"
-		log.Println(errMsg)
-		http.Error(w, errMsg, http.StatusForbidden)
-		return
-	}
-
 	if userID != memberID {
+		if *sessionMember.Pending {
```

Comment:
This checks for if the user making the request is pending, because a pending `admin` for example shouldn't be able to remove a `user` from a conversation.

Reply:
Ohhhh got it. Thanks for the clarification. I keep mixing up the `sessionMember` and `reqMember` lol

Reply:
Should probably change it to be less confusing. Would changing `reqMember` to `reqBodyMember` or just `reqBody` make more sense?

Reply:
Hmm nah it's okay, I think it's good as is. I just mixed it up because the rest of the function gets cut off so I forget haha.

Reply:
Okay, I did change `member` to `targetMember` and `memberID` to `targetMemberID` though because I feel like those were vague.

Reply:
Sounds good. That's actually a lot better!